Modules/TOF/src/TrendingRate.cxx

```diff
@@ -123,29 +128,43 @@ void TrendingRate::computeTOFRates(TH2F* h, std::vector<int>& bcInt, std::vector
       const int bcmax = ibc * 18;
       TH1D* hs = hDiffGlobal.ProjectionY(Form("sign_%d_%d", bcmin, bcmax), ibc, ibc);
       hs->SetTitle(Form("%d < BC < %d", bcmin, bcmax));
+      if (hb->GetBinContent(1) <= 0.) {
+        ILOG(Warning, Support) << "Background bin content is zero, cannot normalize histogram" << ENDM;
+        continue;
+      }
```

Comment:
It is better to move this condition to hback and external to the for loop.
There is no need to check every time inside the loop (note that if you continue inside the loop you have to destroy hs and hb before to continue, outside the loop you don't need to care about that)